src/dataflow/utilities/aws.ts

```diff
@@ -107,6 +107,33 @@ interface ProgramDataPoint{
 
 const allProgramsCache: {[programId: string]: ProgramDataCache} = {};
 
+export const fetchActiveRelays = () => {
```

Comment:
This seems like it could be generalized somewhat, since it shares a lot of code with the function below. OTOH, in the interest of preventing premature optimization, I could also see waiting until a third similar function was added.